src/main/java/de/uni_tuebingen/ub/nppm/model/SelektionBeziehungGemeinschaft.java

```diff
@@ -0,0 +1,10 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import java.util.*;
+
+@Entity
+@Table(name = "selektion_beziehung")
```

Comment:
Actually, the table should have the same name as the class, so if you wanna call the class "SelektionBeziehungGemeinschaft" then the table should be called `selektion_beziehung_gemeinschaft`

Reply:
done